### PR TITLE
Change references to functionmap2 to pkgapi

### PR DESCRIPTION
**Pull request by jrnold**
Since `functionmap2` was renamed to `pkgapi` https://github.com/r-pkgs/pkgapi/commit/b8342cc58a241dbb5ed6f2ccc779a7cb9dd97c18

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -33,6 +33,7 @@ Suggests:
     microbenchmark,
     ggplot2,
     mgcv,
+    pkgapi,
```
Comment (krlmlr) [inline DESCRIPTION]: Why do you want to add it to Suggests: ?

Comment (jrnold) [inline DESCRIPTION]: I thought it was needed. Is it not?

Comment (krlmlr) [inline DESCRIPTION]: roxygen2 isn't there either, pkgapi is required to create the API file only as part of creating the documentation with `devtools::document()`.

**Comment by krlmlr** [comment]
Thanks!

**Pull request merged**